app/actions/sortActions.js

```diff
@@ -0,0 +1,36 @@
+
+import types from 'constants/ActionTypes';
+
+import { CALL_API } from 'redux-api-middleware';
+
+import schemas from 'store/middleware/Schemas';
+import {
+  buildAPIUrl,
+  getErrorTypeDescriptor,
+  getHeadersCreator,
+  getRequestTypeDescriptor,
+  getSuccessTypeDescriptor,
+} from 'utils/apiUtils';
+
+function sortResources(orderBy) {
```

Comment:
Why is sort as its own action/api call? It's only a parameter for the search api call.

Reply:
Corrected this in the new commit. Can you verify if it is okay?